### ISSUE TITLE
Add Filter Absorber notice on raid exit

### ISSUE THREAD

**Issue by thaddeus** [the-hideout/TarkovMonitor#40]
Suggested by Xavier: https://discord.com/channels/668387296973684737/668387297410023435/1178427812303085668

Add an optional feature that reminds the user at raid end to remove the filter absorber from their hideout, thus saving time on the FA (thus money). Similar to the match found, match starting optional notices.

Ideally this would have a counterpart when the user enters the lobby but before starting to match to remind the user to re-install a filter if necessary.

The monitor watches for raid end logs here https://github.com/the-hideout/TarkovMonitor/blob/master/TarkovMonitor/GameWatcher.cs#L160
And triggers an event which is watched for here: https://github.com/the-hideout/TarkovMonitor/blob/042ce2289480a7d25984c3227bcecd86744bd6cb/TarkovMonitor/MainBlazorUI.cs#L470

An example of playing a sound when the option is set for a match found can be seen here: https://github.com/the-hideout/TarkovMonitor/blob/042ce2289480a7d25984c3227bcecd86744bd6cb/TarkovMonitor/MainBlazorUI.cs#L282

**Comment by xaviersand** [comment]
Note for dev: I could also be nice to have the reminder (did you put the filter ON ? when the match starts), and add the sound just after/before the existing reminder "start Failed tasks".

Both before/after raid should be enabled/disabled together with a button in the settings.
